configure-pull-request/action.yml

```
@@ -0,0 +1,49 @@
+name: Configure Pull Request
+
+description: Sets the parameters of an existing Pull Request at once
+
+inputs:
+  github-token:
+    description: A GitHub personal access token with write access to the project
```

```suggestion
    description: A GitHub access token with write access to the project
```
A PAT is something bound to a user account; access tokens would include both PATs and dynamic tokens like the ones obtained with [tibdex/github-app-token](https://github.com/camunda/github-actions-recipes/blob/main/README.md#authenticate-from-a-github-app-via-tibdexgithub-app-token-usages-in-camunda)